src/test/java/org/example/springbootintro/order/OrderRepositoryTest.java

```diff
@@ -0,0 +1,66 @@
+package org.example.springbootintro.order;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+import java.util.Optional;
+import org.example.springbootintro.model.Order;
+import org.example.springbootintro.repository.order.OrderRepository;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith(SpringExtension.class)
+@DataJpaTest
+public class OrderRepositoryTest {
```

Comment:
I can see that you have repository tests, but all test cases are verifying the empty results, insert some data to DB with `@SQL` to test positive results as well, and check other classes where you can use it, especially integrational that I mentioned above